### DISCUSSION TITLE
Captain Talent tree remake to make him more attractive all around

### DISCUSSION THREAD

**Discussion by Swato85** [FakeFishGames/Barotrauma#7382] [Ideas and suggestions]
Sorry from posting in this way, but I was asked/recomended to post it here  on the baro ..  discord, and i am quite tired from writing it down in the first place. I just hope that some if the ideas within get noticed and maybe  adressed even.

I used to play captain a lot. Mainly because of subbuilding/modifying a lot and teaching new guys how what goes so it was kinda natural. Doing a lot of stuff on board and leading expeditions. I was excited when the talents were coming, but once here it feels LIKE A CLEAN SLAP into the face.

Some major issues:

1st talents group
captain is the only class with a talent-forced roleplay, other classes have simply buffs, captain is required to play a drunk.
a) to be drunk to get resistance bonuses
b) to ragdoll around like a drunk to get time-limited damage bonus.

Pro - the new pistol and ammo recipies are nice. But there should be a proper buff asociated to it so that he is not justsomeone with a nice gun.

2nd talents group
The medals are an interesting thing, but fall hugely behind compared to other classes direct buffs.
- you play a lot, u have no issue with experience - the talent should be a subbonus of some real talent.

Additional mark = direct waste of talent. i never had an issue with money (complete missions, loot wrecks, mine - we do all we can on every round and never needed a secondary injections. I would exchange all the money related talents for some expedition related bonuses. Hands on heart you captains out there: Do you like to be more like the Drunk captain from Tin Tin or something like: Kirk, Picard, Phillipe and basicaly any other actualy usefull captain anywhere/ not just sci-fi.

Even basic small buffs on expedition would be a lot better.
I. Preparadness - First damage on away mission is decreased by 80% for captain and personel in his area of influence.
II. Stay sharp - (slapping the Drunk image back isnt it? :-D) - Schock/Psychosis impact on team in influence area is decreased.
III. We stand, for there is hope! - captain and personel in his area of influence are capable to stay on foot an extra 30 seconds after point when they should fall unconscious or be otherwise incapacitated.
IV. There is no easy path - (Captain allways expects complications but stays prepared), decrease damage of traps, maybe delay their trigger time, some directional identification of possible danger (traps, enemies - you can even make it 50% effective so sometimes it gives false alarms, but keeps everybody prepared = that is the point.

Some minor additions to this tree:
His damage bonus to monsters on board should be for all enemies on board - even against uprisals.
Not sure how the bonus for the sinking sub is working, but it definitely should not be time based but instead stay active for as long as the sub remains in this critical state.
- and propably get some fade time when it gets out if the critial state.

3rd. talents group.
all around terrible
- basic fact: 4 of 6 talents give nothing to captain, maybe even all six as the description talks about captains crew, not him.
Very bad design in my opinion - captain gets nothing and canonly trust that whoever gets the bonus buff actually puts it to good use -no guarantee of course.

Simple change here just like in the real world: Lead by an example! Not by a payed or otherwise cheaply bestowed title. Personaly I dont care for ranks of anybody unless I see that he really earned it, that he really is capable in the position he is in!
- so some basic change here, even if it would mean to decrease the bonuses - give the
bonus to captain and the buffed crewmembers so that there can at least be seen where the inspiration comes from.
Secondly I would propably admire alot more an on board buff, so that all crew on board get some kinda of buff as long as the captain is on board too not a single guys inspiring jelausy and hatred among crew. Very bad method of leadership!.

The health bonus is propably the only usefull thing here, although if it does not apply to captain, the whole tree stays useless.

Some other points to highlight.
There are no sonar related talents
There are no pathing/navigation related talens
There are no talents adressing captains capability to steer the sub in general (e.g.: more effective speads, collision damage decrease, etc....)
Given the area where we are, talents adressings sub stealthiness are also missing (e.g.: near invisibility to wild life when sub sits on the ground, higher engine effectivity limit for staying in the "silent mode" and similar.

**Comment by gerakis** [comment]
I agree that the left talent tree (the pistol tree) is pretty garbage. The physicorium for the ammo was particularly awful. The middle and right tree are ok. The extra mission limit at the bottom of each tree is massively good, and also stacks if you have 2 captains on board.

I also agree that talents for driving and sonar would be good.
